src/plugins/newsfeed/types.ts

```diff
@@ -19,16 +19,14 @@
 
 import { Moment } from 'moment';
 
-export interface NewsfeedPluginInjectedConfig {
-  newsfeed: {
-    service: {
-      urlRoot: string;
-      pathTemplate: string;
-    };
-    defaultLanguage: string;
-    mainInterval: number; // how often to check last updated time
-    fetchInterval: number; // how often to fetch remote service and set last updated
+// Ideally, we may want to obtain the type from the configSchema and exposeToBrowser keys...
+export interface NewsfeedPluginBrowserConfig {
+  service: {
+    urlRoot: string;
+    pathTemplate: string;
   };
+  mainInterval: number; // how often to check last updated time
```

Comment:
hm... It should be `Duration`
```js
import { Duration } from 'moment';
```

Reply:
Changed! I had to wrap the browser-exposed Duration config in `moment.duration` because, apparently, the `exposeToBrowser` config is stringified (which makes sense). Maybe it's worth the KP team to explore a way to revert the stringifying before passing it over to the plugin?